src/main/java/org/jabref/logic/search/SearchQuery.java

```diff
@@ -58,13 +58,15 @@ String format(String regex) {
     private final String query;
     private final boolean caseSensitive;
     private final boolean regularExpression;
+    private final boolean fulltext;
     private final SearchRule rule;
 
-    public SearchQuery(String query, boolean caseSensitive, boolean regularExpression) {
+    public SearchQuery(String query, boolean caseSensitive, boolean regularExpression, boolean fulltext) {
```

Comment:
I know, they weren't enums before, but as the SearchQuery is used a few times, would it be possible to quickly make enums of the bools?